library/evaluation/calibration.py

```diff
@@ -0,0 +1,39 @@
+from pycaleva import CalibrationEvaluator
```

Comment:
This is really cool! It gives info on how our classification labels are distributed?